lib/jekyll/configuration.rb

```diff
@@ -226,9 +226,10 @@ def backwards_compatibilize
       # Provide backwards-compatibility
       check_auto(config)
       check_server(config)
+      check_plugins(config)
 
       renamed_key "server_port", "port", config
-      renamed_key "plugins", "plugins_dir", config
+      renamed_key "gems", "plugins", config, false
```

Comment:
Based on the presence of `false`, this could just be `config["gems"] = config["plugins"] if config.key?("plugins")`